runtime/core/portable_type/c10/c10/targets.bzl

```diff
@@ -26,6 +26,7 @@ def define_common_targets():
             "util/TypeSafeSignMath.h",
             "util/bit_cast.h",
             "util/floating_point_utils.h",
+            "util/irange.h",
```

Comment:
is irange.h not part of headers from pytorch? or we decided that is not worth sharing?

Reply:
it is a pytorch header, but it is used in executorch core. therefore it must be copied over just like everything else in runtime/core/portable_type/c10/c10 to keep the build hermetic for embedded users